lib/icinga/apievents.cpp

```diff
@@ -30,6 +30,9 @@ void ApiEvents::StaticInitialize()
 	Downtime::OnDowntimeRemoved.connect(&ApiEvents::DowntimeRemovedHandler);
 	Downtime::OnDowntimeStarted.connect(&ApiEvents::DowntimeStartedHandler);
 	Downtime::OnDowntimeTriggered.connect(&ApiEvents::DowntimeTriggeredHandler);
+
+	ConfigObject::OnActiveChanged.connect(&ApiEvents::ObjectCreatedOrDeletedHandler);
+	ConfigObject::OnVersionChanged.connect(&ApiEvents::ObjectUpdatedHandler);
```

Comment:
Name these not after what they're doing, but after the events they're connected to.